react-app/components/FileManager/index.jsx

```diff
@@ -58,6 +58,41 @@ const FileManager = (props) => {
     downloadFileHook(fullUrl, display_name);
   };
 
+  const downloadFiles = (filesToDownload) => {
+    downloadFilesTemp(filesToDownload)
+  };
+
+  const downloadFilesTemp = async (filesToDownload) => {
```

Comment:
why is this `Temp`?

Reply:
It's a only a wrapper, naming is hard. Any ideas?